Content/Library/README.md

```diff
@@ -44,14 +43,59 @@ $ ./build.sh  // on unix
 - `DISABLE_COVERAGE` Will disable running code coverage metrics.  AltCover can have [severe performance degradation](https://github.com/SteveGilham/altcover/issues/57) so it worth disabling when looking to do a quicker feedback loop.
   - `DISABLE_COVERAGE=1 ./build.sh`
 
-### Watch Tests
 
-The `WatchTests` target will use [dotnet-watch](https://github.com/aspnet/Docs/blob/master/aspnetcore/tutorials/dotnet-watch.md) to watch for changes in your lib or tests and re-run your tests on all `TargetFrameworks`
```

Comment:
s/will use/uses. It's not conditional, correct?

Reply:
Should add it matches the voice used elsewhere as well.

Reply:
Removed this section entirely so shouldn't be a problem

Reply:
You know, it's red, this should have been obvious. Ignore me.